support/mock/mockrt.go

```diff
@@ -207,7 +207,7 @@ func (rt *Runtime) Send(toAddr addr.Address, methodNum abi.MethodNum, params abi
 		rt.Abort(exitcode.SysErrorIllegalActor, "side-effect within transaction")
 	}
 	if len(rt.expectSends) == 0 {
-		rt.t.Errorf("unexpected expectedMessage to: %v method: %v, value: %v, params: %v", toAddr, methodNum, value, params)
+		rt.t.Fatalf("unexpected expectedMessage to: %v method: %v, value: %v, params: %v", toAddr, methodNum, value, params)
```

Comment:
No need to be fatal?

Reply:
If it's not fatal all the code below it will panic, but if we want this as an error I can modify the below to accommodate it.